src/next/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,331 @@
+import {
+  type HeroGradientProps,
+  type HeroSplitProps,
+  type HeroFloatingProps,
+  type HeroCopyLedProps,
+  type HeroFloatingImageProps,
+} from "~/common/Hero"
+import Button from "~/next/components/Button"
+
+export type HeroProps =
+  | HeroGradientProps
+  | HeroSplitProps
+  | HeroFloatingProps
+  | HeroCopyLedProps
+  | HeroFloatingImageProps
```

Comment:
Honestly this one I'm not too sure. Ideally we should make sure that they have a common prop to reference, but it seems like Next and Classic themes for the Hero banner has deviated quite a bit. Opting to separate for now until we figure out more details about the state of the hero banners (maybe backport to Classic so that both sides can have the same variants).

Reply:
Maybe can find the commonality between Next and Classic, and for Next can extend from this base props? I don't think we can achieve 100% compat bet the 2 templates as Next is meant to be sort of "greenfield" approach in building components

Reply:
Hmm I'm thinking of keeping them separate for now. From what it seems, Hero component is the only component that suffers from this problem, and I don't have sufficient guarantees on how much the Hero component will change in the future. I'm opting to revisit this in the future once we finalise the designs and prepare for larger scale rollout.